packages/babel-preset/__fixtures__/assets/code.ts

```diff
@@ -6,6 +6,12 @@ import empty from './empty.jpg';
 
 export const useStyles = makeStyles({
   rootA: { backgroundImage: `url(${blank})` },
-  rootB: { backgroundImage: `url(${blankDuplicate})` },
+  rootB: { backgroundImage: `url("${blankDuplicate}")` },
   rootC: { backgroundImage: `url(${empty})` },
+  httpUrl: { backgroundImage: 'url(http://www.example.com)' },
+  httpsUrl: { backgroundImage: 'url(https://www.example.com)' },
+  httpsUrlWithQuotes: { backgroundImage: 'url("https://www.example.com")' },
+  dataUrl: { backgroundImage: 'url(data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7)' },
+  hashOnly: { filter: 'url(#a)' },
```

Comment:
Can you please move out assets that are not "extractable" to a separate fixture? I.e. create `packages/babel-preset/__fixtures__/assets-urls/code.ts`, `packages/babel-preset/__fixtures__/assets/output.ts` and add to `packages/babel-preset/src/transformPlugin.test.ts`?

Reply:
Tests are separated as #237 does.